artifacts/deploy/kube-controller-manager.yaml

```diff
@@ -37,17 +37,17 @@ spec:
             - --authentication-kubeconfig=/etc/kubeconfig
             - --authorization-kubeconfig=/etc/kubeconfig
             - --bind-address=0.0.0.0
-            - --client-ca-file=/etc/karmada/pki/server-ca.crt
+            - --client-ca-file=/etc/karmada/pki/ca.crt
             - --cluster-cidr=10.244.0.0/16
             - --cluster-name=karmada
-            - --cluster-signing-cert-file=/etc/karmada/pki/server-ca.crt
-            - --cluster-signing-key-file=/etc/karmada/pki/server-ca.key
+            - --cluster-signing-cert-file=/etc/karmada/pki/ca.crt
+            - --cluster-signing-key-file=/etc/karmada/pki/ca.key
```

Comment:
Just my question, we don't seem to have `server-ca.key/ca.key` in the `karmada-cert-secret` secret from the beginning.